CMakeLists.txt

```diff
@@ -13,6 +13,18 @@ option(ENABLE_QT "Enable Qt (use the experimental Qt interface)" OFF)
 option(ENABLE_PCH "Use PCH to speed up compilation" ON)
 option(ENABLE_LTO "Enables Link Time Optimization" OFF)
 option(ENABLE_GENERIC "Enables generic build that should run on any little-endian host" OFF)
+
+# Enable SDL for default on operating systems that aren't OSX, Android, Linux or Windows.
+if(NOT APPLE AND NOT ANDROID AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND NOT MSVC)
+	option(ENABLE_SDL "Enables SDL as a generic controller backend" ON)
+else()
+	option(ENABLE_SDL "Enables SDL as a generic controller backend" OFF)
+endif()
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+	option(ENABLE_EVDEV "Enables the evdev controller backend" ON)
+endif()
```

Comment:
Turns out it depends on which version of cmake is used. On the older 2.8 version which the android buildbot is using Android is Linux. Fixing now.